hbase-protocol-shaded/src/main/protobuf/server/coprocessor/Aggregate.proto

```diff
@@ -48,6 +49,8 @@ message AggregateResponse {
    */
   repeated bytes first_part = 1;
   optional bytes second_part = 2;
+  optional bytes  next_chunk_start_row = 3;
+  optional uint64 wait_interval_ms     = 4 [default = 0];
```

Comment:
Can we save 4 bytes over the wire by using an `uint32` instead? In reality, probably `uint16` is enough to represent a realistic wait interval, but `uint32` gives us a lot more breathing room.

Reply:
In terms of what's realistic, I agree 32 or even 16 bits is fine. I picked 64 bits because RpcThrottlingException exposes the desired wait time as a `long`, and I didn't want to clobber any information from it.